packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Statements/SwitchCaseStatement.cs

```diff
@@ -12,16 +12,18 @@ public sealed class SwitchCaseStatement : MethodBodyStatement
     {
         public IReadOnlyList<ValueExpression> Matches { get; }
         public MethodBodyStatement Statement { get; }
+        public bool AddBraces { get; }
 
-        public SwitchCaseStatement(IReadOnlyList<ValueExpression> matches, MethodBodyStatement statement)
+        public SwitchCaseStatement(IReadOnlyList<ValueExpression> matches, MethodBodyStatement statement, bool addBraces = false)
         {
             Matches = matches;
             Statement = statement;
+            AddBraces = addBraces;
         }
 
-        public SwitchCaseStatement(ValueExpression match, MethodBodyStatement statement) : this([match], statement) { }
+        public SwitchCaseStatement(ValueExpression match, MethodBodyStatement statement, bool addBraces = false) : this([match], statement, addBraces) { }
```

Comment:
I don't think we need the add braces bool here.

If you are using for this
![image](https://github.com/microsoft/typespec/assets/64171366/129185ce-1ab4-4c6a-8cc4-9e00831757af)
the braces are not required and its common to not have them.